packages/testing/config/jest/jest-preset.js

```diff
@@ -1,4 +1,6 @@
-// used by cli `rw test` command
```

Comment:
Because this file is renamed should this be considered breaking change? CRWA template doesn't refer to this file and I don't think it's documented to use directly. But it is possible that someone has extended this file if they created a root level jest.config.js. What do you all think?

Reply:
I don't think this will be an issue!